Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -83,18 +84,25 @@ public extension MoyaProvider {
             switch stubBehavior {
             case .never:
                 switch target.task {
```

Comment:
Okay, I think I understand your issue now. I've just extracted the inner switch statement to a method named `unstubbedRequest` in ab18763cfd7f70a3057b6c9af4c21871c380b21a. I hope this is what you were expecting.